app/test/integration_test.go

```diff
@@ -63,44 +64,16 @@ func (s *IntegrationTestSuite) SetupSuite() {
 func (s *IntegrationTestSuite) TestMaxBlockSize() {
 	t := s.T()
 
-	singleBlobTxGen := func(c client.Context) []coretypes.Tx {
-		return blobfactory.RandBlobTxsWithAccounts(
-			s.ecfg,
-			random.New(),
-			s.cctx.Keyring,
-			c.GRPCClient,
-			600*kibibyte,
-			1,
-			false,
-			s.accounts[:20],
-		)
-	}
-
-	// This tx generator generates txs that contain 3 blobs each of 200 KiB so
-	// 600 KiB total per transaction.
-	multiBlobTxGen := func(c client.Context) []coretypes.Tx {
-		return blobfactory.RandBlobTxsWithAccounts(
-			s.ecfg,
-			random.New(),
-			s.cctx.Keyring,
-			c.GRPCClient,
-			200*kibibyte,
-			3,
-			false,
-			s.accounts[20:40],
-		)
-	}
-
 	randomTxGen := func(c client.Context) []coretypes.Tx {
 		return blobfactory.RandBlobTxsWithAccounts(
 			s.ecfg,
 			random.New(),
 			s.cctx.Keyring,
 			c.GRPCClient,
-			50*kibibyte,
+			mebibyte-1,
 			8,
 			true,
-			s.accounts[40:120],
+			s.accounts[:60],
 		)
 	}
```

Comment:
I think we were a bit overzealous or rushed with a few of these tests before mainnet and should refactor them